### ISSUE TITLE
Accuracy Difference for inference compared to benchmark results

### ISSUE THREAD

**Issue by shekarneo** [PaddlePaddle/PaddleOCR#6683]
请提供下述完整信息以便快速定位问题/Please provide the following information to quickly locate the problem

- 系统环境/System Environment：ubuntu 20.04/CPU
- 版本号/Version：Paddle：2.2.2  PaddleOCR：2.5 
-  问题相关组件/Related components：i tried to export EAST, SAST detectors to inference and tested with few images but compared to DB net the accuracy is less but in benchmark the accuracy is high, same thing for SVTR_Tiny. 

https://github.com/PaddlePaddle/PaddleOCR/blob/release/2.5/doc/doc_en/algorithm_overview_en.md#11

**Comment by andyjpaddle** [comment]
maybe the training set is different with your test set, you can fine-tune the model with your own dataset
